form/src/App.js

```diff
@@ -29,28 +36,27 @@ function App() {
     showModal();
   };
 
-  const CustomSelect = function (props) {
-    
-    return (
-      <Select
-        id="input"
-        className="basic-single"
-        classNamePrefix="select"
-        options={props.options.enumOptions}
-        placeholder={props.label}
-        isSearchable={true}
-        onChange={async (e) => {
-          await props.onChange(e.value);
-        }}
-      ></Select>
-    );
-  };
-  
-  const widgets = {
-    SelectWidget: CustomSelect,
-  };
+  // const CustomSelect = function (props) {
+
+  //   return (
+  //     <Select
+  //       id="input"
+  //       className="basic-single"
+  //       classNamePrefix="select"
+  //       options={props.options.enumOptions}
+  //       placeholder={props.label}
+  //       isSearchable={true}
+  //       onChange={async (e) => {
+  //         await props.onChange(e.value);
+  //       }}
+  //     ></Select>
+  //   );
+  // };
+
+  // const widgets = {
+  //   SelectWidget: CustomSelect,
+  // };
```

Comment:
You're mixing things within a single commit again. These changes are unrelated but you still commit them in a single run, resulting in one PR isn't compatible with the other. Use three PRs instead and if they depend on one another, mention that in the description.

Reply:
Instead of *commenting* the code, please remove it. It'll still be in the git history to be looked at for future reference.